README.md

```diff
@@ -1,41 +1,41 @@
 # Atleta Network
 
-The project is built on a [frontier template](https://github.com/PureStake/frontier) —
-[FRAME](https://docs.substrate.io/v3/runtime/frame)-based
-[Substrate](https://substrate.io) node with the Ethereum RPC support, ready for
+The project is built on a [frontier template](https://github.com/polkadot-evm/frontier) —
+[FRAME](https://docs.substrate.io/reference/)-based
+[Substrate](https://substrate.io) node with the [Ethereum RPC](https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods) support, ready for
 hacking :rocket:
 
 
 
 
 ## Build & Run
 
-#### Build with "fast-runtime" feature
-To build the test/dev network with the "fast-runtime" feature, execute the following command from the project root:
-```
-$ cargo build --release --features "fast-runtime" 
+#### Build with `fast-runtime` feature
+To build the _testnet_/_devnet_ network with the `fast-runtime` feature, execute the following command from the project root:
+```shell
+cargo build --release --features "fast-runtime" 
 ```
 
-#### Build without "fast-runtime" feature
-To build the mainnet network execute the following command from the project root:
+#### Build without `fast-runtime` feature
+To build the _mainnet_ network execute the following command from the project root:
 
-```
-$ cargo build --release
+```shell
+cargo build --release
 ```
 
 
-To execute the chain, run:
+To execute the _devnet_ chain, run:
 
-```
-$ ./target/release/atleta-node --dev
+```shell
+./target/release/atleta-node --dev
 ```
 
 
 
 
 ## Genesis Configuration
 
-In order to view an EVM account, use the `Developer` tab of the Polkadot UI
+In order to view an EVM account on _devnet_, use the [`Developer`](https://polkadot-explorer.atleta.network/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/settings/developer) tab of the Polkadot UI
```

Comment:
ws://127.0.0.1:9944 - here a local node is expected to be running. Also, we can't say what kind of the network the user runs.

Reply:
Done.
Just for note that mentioned `Developer` tab is not available on [_testnet_ explorer](https://polkadot-explorer.atleta.network/).

Reply:
hmm, probably this instruction is outdated then

Reply:
Still actual on polkadot.js.org/apps. Just our explorer fork hides this tab by some reason.

Reply:
ah ok. that's because we've [pre-configured](https://github.com/Atleta-network/polkadot-js-apps/blob/3c5c6f845c884cedc46b013671434c140c7f5d18/packages/apps-config/src/api/spec/atleta.ts#L12) it in our fork. so the instruction is relevant only to polkadot.js.org/apps.